src/lazy/encoder/annotate.rs

```diff
@@ -18,11 +18,11 @@ pub trait Annotatable {
     ///# use ion_rs::IonResult;
     ///# fn main() -> IonResult<()> {
     /// use ion_rs::{Element, IonData};
-    /// use ion_rs::lazy::encoder::text::LazyRawTextWriter_1_0;
+    /// use ion_rs::lazy::encoder::text::v1_0::writer::LazyRawTextWriter_1_0;
```

Comment:
🗺️ There are now `v1_0` and `v1_1` directories for the respective text writer implementations.